meta-oe/recipes-printing/qpdf/qpdf_9.1.0.bb

```diff
@@ -0,0 +1,33 @@
+DESCRIPTION = "PDF transformation/inspection software"
+HOMEPAGE = "http://qpdf.sourceforge.net"
+LICENSE = "Artistic-2.0"
+SECTION = "libs"
+DEPENDS = "libpcre zlib libjpeg-turbo"
+
+SRC_URI = "https://github.com/qpdf/qpdf/archive/release-qpdf-${PV}.tar.gz"
```

Comment:
I see 
`SRC_URI uses unstable GitHub archives [src-uri-bad]`
perhaps its better to use 
```
SRCREV = "some sha"
SRC_URI = "git://github.com/qpdf/qpdf;protocol=https"
S = "${WORKDIR}/git"
```

Reply:
Ok, there are tarballs at sourceforge, I will use those.